### DISCUSSION TITLE
Keep empty lines/formatting in compose and env files

### DISCUSSION THREAD

**Discussion by ChristopherHaws** [louislam/dockge#549] [Feature Request]
### ⚠️ Please verify that this feature request has NOT been suggested before.

- [X] I checked and didn't find similar feature request

### 🏷️ Feature Request Type

UI Feature

### 🔖 Feature description

Currently when I save my compose and env files using the web ui, all empty lines are removed which makes it harder to read large compose files. I would rather remove the lines myself if I want them removed, the UI should simply save exactly what a paste into the textbox.

### ✔️ Solution

The UI should save the compose and env file's exactly as they are pasted into the textbox

**Comment by david-christ** [comment]
> the UI should simply save exactly what a paste into the textbox

I would love to have this too. It goes beyond removing empty lines. Dockge also adds a networking section and does other formatting/post-processing. I would like to be able to turn _all_ of that off.

**Comment by SootyOwl** [comment]
I agree with this - I also had certain comments removed from the file which has been quite annoying as they were very useful.

**Comment by truncsphere** [comment]
I'm running into this as well. Upvoted.
